AdyenCardScanner/Sources/CardScanner.swift

```diff
@@ -7,9 +7,11 @@
 import Foundation
 import UIKit
 
+public typealias CardDetails = (number: String?, expirationDate: Date?)
```

Comment:
We have a `CardDetails` struct, this may collide with it

Reply:
It's a framework, and its usage would be strictly limited to one file `CardScannerController.swift` and I will just use namespace prefix for that matter. Do you think the rest of the existing occurrences would conflict with this one that is not imported in those files?

Reply:
I can rename it into `CardData` or `CardScanDetails` instead

Reply:
Renamed to `CardScanDetails`. This way we avoid any possibility of collision.